hello-world.md

```diff
@@ -117,17 +117,41 @@ We're declaring some variables with the syntax `varName := value`, which lets us
 
 We are calling the `Errorf` _method_ on our `t`, which will print out a message and fail the test. The `f` stands for format, which allows us to build a string with values inserted into the placeholder values `%q`. When you make the test fail, it should be clear how it works.
 
-You can read more about the placeholder strings in the [fmt go doc](https://golang.org/pkg/fmt/#hdr-Printing). For tests, `%q` is very useful as it wraps your values in double quotes.
+You can read more about the placeholder strings in the [fmt documentation](https://pkg.go.dev/fmt#hdr-Printing). For tests, `%q` is very useful as it wraps your values in double quotes.
 
 We will later explore the difference between methods and functions.
 
-### Go doc
+### Go's documentation
 
-Another quality-of-life feature of Go is the documentation. You can launch the docs locally by running `godoc -http=localhost:8000`. If you go to [localhost:8000/pkg](http://localhost:8000/pkg), you will see all the packages installed on your system.
+Another quality-of-life feature of Go is the documentation.  We just saw the documentation for the fmt package at the official package viewing website, and Go also provides ways for quickly getting at the documentation offline.
 
-The vast majority of the standard library has excellent documentation with examples. Navigating to [http://localhost:8000/pkg/testing/](http://localhost:8000/pkg/testing/) would be worthwhile to see what's available to you.
+Go has a built-in tool, doc, which lets you examine any package installed on your system, or the module you're currently working on. To view that same documentation for the Printing verbs:
+
+```
+$ go doc fmt
+package fmt // import "fmt"
+
+Package fmt implements formatted I/O with functions analogous to C's printf and
+scanf. The format 'verbs' are derived from C's but are simpler.
+
+# Printing
+
+The verbs:
+
+General:
+
+    %v	the value in a default format
+    	when printing structs, the plus flag (%+v) adds field names
+    %#v	a Go-syntax representation of the value
+    %T	a Go-syntax representation of the type of the value
+    %%	a literal percent sign; consumes no value
+...
+```
+
+Go's second tool for viewing documentation is the pkgsite command, which powers Go's official package viewing website.  You can install pkgsite with `go install golang.org/x/pkgsite/cmd/pkgsite@latest`, then,  `pkgsite -open .`.
```

Comment:
At least in my case, when I follow the steps in this part, the `pkgsite -open .` simply returns

```zsh: command not found: pkgsite```

Reply:
Hmm, do you not have ~/go/bin on your PATH? Or, did you need to restart your shell session?

Reply:
I didn't. I will re-check if this tutorial or Go docs mention that. If not, I think we also need to add it somewhere in learn-go-with-tests.

After adding the ~/go/bin to the PATH, yep, it works as expected - my issue was only with relying on possibly missing instructions.

Reply:
I grepped learn-go-with-tests for PATH - didn't find any information about adding ~/go/bin to the $PATH.

With the official Golang docs, I've only found [this note in the installation process](https://go.dev/doc/install), asking to add /usr/local/go/bin to $PATH, a[s well as to add $HOME/go to $GOPATH](https://go.dev/wiki/SettingGOPATH), which is not enough for my setup to make `pkgsite` just work like in the example. 

Can we at least add a short note here, something along the lines of 

"If, even after the installation, your system doesn't recognize the `pkgsite` command, make the binaries from your $GOPATH are added to the $PATH" ?

Reply:
I was wrong about the Go installer doing that for me/us, sorry about that @Battlesheepu.

Starting at this issue, https://github.com/golang/go/issues/30268, and following a link to https://go.dev/doc/code#Command, I finally found some official documentation about modifying $PATH.

Yeah, I'll see where I want to add some direction about checking $PATH. Thanks! :)

Reply:
No need to apologize - you're an angel.

Thanks for putting the hard work in!

Reply:
Added notes about go install, install directories, and $PATH.